### DISCUSSION TITLE
chrome://settings/signOut is blocked when trying to use killcurly

### DISCUSSION THREAD

**Discussion by CALR3F** [3kh0/ext-remover#499] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
It isn't blocked by an extension it says "The person who set up this computer has chosen to block this site.
Try contacting the system admin." Is there any way to get around this? None of the other methods that work for the current update can disable GoGuardian so this is my only hope.

**Comment by pazza2234** [comment, marked as answer]
well, that sucks. you nead the killcurly to use the killcurly. without other methods of disabling exstentions there is not mutch you can do. And at that point you wont nead the killcurly.

**Reply by Aka-but-nice** [reply]
@CALR3F I don't know if you have discord, but if you do, I will send you the other link for this

**Reply by CALR3F** [reply]
Srry I don't have discord, can you just put the link here?

**Reply by Aka-but-nice** [reply]
MAN WHY DID YOUY POST IT

**Reply by Aka-but-nice** [reply]
@3kh0 Ban this dude, this link was private Bypassi said he didn't want it public

**Reply by Aka-but-nice** [reply]
I deleted the message
